src/components/common/ClassroomCard.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const ClassroomCard = ({ isSelected, classroomDetails, classroomNo }) => {
+	const [selected, setSelected] = useState();
+	const onChangeCard = (e) => {
+		const target = e.target;
+		if (target.checked) {
+			setSelected(target.value);
+			return classroomDetails;
```

Comment:
By returning the data from here I mean, returning the data in a callback fn prop.
```suggestion
		 onCardSelection(classroomDetails);
```

Reply:
resolved